_fuel_dev_environment/docker-compose.yml

```diff
@@ -44,6 +42,19 @@ services:
       # Env vars for the fuel core script.
       L1_CHAIN_HTTP: http://l1_chain:8545
       DEPLOYER_ADDRESSES_PATH: http://deployer:8081/addresses.json
+      CONSENSUS_KEY_SECRET: 'a449b1ffee0e2205fa924c6740cc48b3b473aa28587df6dab12abc245d1f5298'
```

Comment:
I guess this has to be hardcoded since it's yaml. Is it safe to have this secret here?

Reply:
This key means nothing outside of this dev environment. It can be set to any value and this one was chosen at random.